app/models/user.rb

```diff
@@ -41,8 +41,10 @@ class User < ApplicationRecord
   delegate   :super_admin_user?, :request_admin_user?, :self_service?, :limited_self_service?, :report_admin_user?, :only_my_user_tasks?,
              :to => :miq_user_role, :allow_nil => true
 
-  validates_presence_of   :name, :userid
-  validates :userid, :unique_within_region => {:match_case => false}
+  validates :name, :presence => true, :length => {:maximum => 100}
+  validates :first_name, :length => {:maximum => 100}
+  validates :last_name, :length => {:maximum => 100}
+  validates :userid, :presence => true, :unique_within_region => {:match_case => false}
```

Comment:
I believe you pulled in https://github.com/ManageIQ/manageiq/pull/21401 accidentally.  

Maybe you want to do `git pull --rebase upstream master` or add this to you ~/.gitconfig:

```
[pull]
rebase = true
```